src/ducks/transactions/TransactionRow.jsx

```diff
@@ -149,7 +149,7 @@ class _RowMobile extends React.PureComponent {
     }
 
     return (
-      <List.Row onRef={onRef} {...rowRest}>
+      <List.Row onRef={onRef} {...rowRest} onClick={this.handleSelect}>
```

Comment:
It doesn't cause any problem to open the categories modal when clicking on the category icon?

Reply:
I removed the commit, I should not have commited this, it is too risky.